src/services/message-formating/message-formatting.service.ts

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash"
+
+export class MessageFormattingService {
+  private static _instance: MessageFormattingService
+
+  public static getInstance(): MessageFormattingService {
+    if (_.isNil(MessageFormattingService._instance)) {
+      MessageFormattingService._instance = new MessageFormattingService()
+    }
+    return MessageFormattingService._instance
+  }
+
+  public format(formatToApply: string, text: string) {
```

Comment:
formatToApply should be the enum type instead of a string.
This is more precise.

Reply:
Ok, I will change the type into the Enum.